master/src/main/java/org/apache/celeborn/service/deploy/master/SlotsAllocator.java

```diff
@@ -219,7 +219,7 @@ private static StorageInfo getStorageInfo(
                 diskInfos[diskIndex].storageType(),
                 availableStorageTypes);
         diskIndex = (diskIndex + 1) % diskInfos.length;
```

Comment:
Just deleting line 221 will be better and the code style will be consistent with the rest.